File: AccuPay/Domain/Paystubs/DivisionMinimumWage.vb
Option Strict On

Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Global.AccuPay.Entity

    <Table("divisionminimumwage")>
    Public Class DivisionMinimumWage

        <Key>
        <DatabaseGenerated(DatabaseGeneratedOption.Identity)>
        Public Property RowID As Integer?

        Public Property OrganizationID As Integer?

        Public Property DivisionID As Integer?

        Public Property Amount As Decimal

        Public Property EffectiveDateFrom As Date

        Public Property EffectiveDateTo As Date

    End Class

End Namespace

File: Accupay.Data/Entities/DivisionMinimumWage.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AccuPay.Data.Entities
{
    [Table("divisionminimumwage")]
    public class DivisionMinimumWage
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int? RowID { get; set; }

        public int? OrganizationID { get; set; }

        public int? DivisionID { get; set; }

        public decimal Amount { get; set; }

        public DateTime EffectiveDateFrom { get; set; }

        public DateTime EffectiveDateTo { get; set; }
    }
}